architecture/2024-08-bicepdeployment-controller.md

```diff
@@ -0,0 +1,693 @@
+# Radius BicepDeployment Controller
+
+* **Author**: Will Smith (@willdavsmith)
+
+## Overview
+
+<!--
+Provide a succinct high-level description of the component or feature and 
+where/how it fits in the big picture. The overview should be one to three 
+paragraphs long and should be understandable by someone outside the Radius
+team. Do not provide the design details in this, section - there is a
+dedicated section for that later in the document.
+-->
+
+Today, users of Radius and future adopters of Radius use many Kubernetes-specific tools in their production workflows. Most of these tools operate on Kubernetes resources exclusively - which presents a problem when trying to deploy resources defined in Bicep manifests. This design proposes the creation of a new Kubernetes reconciler in Radius called the BicepDeployment Reconciler that will allow users to deploy resources defined in Bicep manifests using Kubernetes tooling.
+
+## Today's Status
+
+Today, Radius users are unable to deploy resources defined in Bicep manifests using Kubernetes tooling. This is because Kubernetes tools do not understand Bicep manifests. Users must use the Radius CLI to deploy resources defined in Bicep manifests.
+
+## Terms and definitions
```

Comment:
Defining what we mean by Kubernetes tools would be helpful. It doesn't have to be an exhaustive list, but listing their commonalities would help.